js/ui/radio_group/radio_group.js

```diff
@@ -107,7 +107,7 @@ class RadioCollection extends CollectionWidget {
         return this._itemContainer().children(this._itemSelector());
     }
 
-    _setAriaSelected() {}
+    _setAriaSelectionAttribute() {}
```

Comment:
It can be rewrite like `_setAriaSelectionAttribute: noop` everywhere

Reply:
i did not refactor anything here, just renamed the function